mjpc/tasks/allegro/allegro.cc

```diff
@@ -0,0 +1,101 @@
+#include "mjpc/tasks/allegro/allegro.h"
+#include <iostream>
+
+#include <cmath>
+#include <string>
+#include <vector>
+
+#include <mujoco/mujoco.h>
+#include "mjpc/task.h"
+#include "mjpc/utilities.h"
+
+namespace mjpc {
```

Comment:
we need to format this file from here down (the includes above are already correct). I am using the vscode extension 'Clang-Format'. in the settings the style should be set to google.